server/bin/gold/test-7.4.txt

```diff
@@ -83,12 +83,12 @@ len(actions) = 5
             "@generated-by": "70015f01dedbadbeefc105edcafedead",
             "@metadata": {
                 "controller_dir": "alphaville",
-                "file-date": "2019-03-07T02:10:59",
-                "file-name": "/var/tmp/pbench-test-server/test-7.4/pbench/archive/fs-version-001/alphaville/test-7.4.tar.xz",
-                "file-size": 1484,
-                "md5": "90c3fceba53c859d87ad66edfcc1154e",
+                "file-date": "2020-05-05T02:38:54",
+                "file-name": "/var/tmp/pbench-test-server/test-7.4/pbench/archive/fs-version-001/alphaville/test_7.4_2015.09.21T15.31.08.tar.xz",
```

Comment:
Just to test my understanding: the renaming of the tarball to include the date is *NOT* necessary for this PR. You are doing it because you are looking ahead to the culling PR - correct?

Reply:
Correct.

None of this is necessary by itself.  But the amount of change that results swamps future changes.

So by making all these test environment changes here, we NO other code changes, we isolate the massive amount of change to make things easier in the future.

Reply:
Do you care that e.g. in the 5.2 tar ball, there are internal tar balls with dates like 1970-01-01T00:00:00 (i.e. no 42)? I presume these will not cause problems but just wanted to point them out.

Reply:
I think most, if not all the tar balls, don't have 42 on them.  The 42 was for the log file entries.  I may not have been perfectly consistent on that.